### ISSUE TITLE
Installation instruction

### ISSUE THREAD

**Issue by paneq** [RailsEventStore/rails_event_store-rspec#2]
Currently we have:

Add this line to your application's Gemfile:

```ruby
gem 'rails_event_store-rspec'
```

Shouldn't we advise to use:

```
group :test do
  gem 'rails_event_store-rspec'
end
```

??

**Comment by pawelpacana** [comment]
Probably yes, there's no use outside tests for it.

**Comment by pawelpacana** [comment]
Fixed in 3443a3efa2ec543b96189376363a8e665281f07d.
